Mlem/API/APIClient/APIClient.swift

```diff
@@ -259,8 +259,13 @@ extension APIClient {
     
     // MARK: - User requests
 
-    func deleteUser(user: SavedAccount, password: String) async throws {
-        let request = DeleteAccountRequest(account: user, password: password)
+    func legacyDeleteUser(user: SavedAccount, password: String, deleteContent: Bool) async throws {
```

Comment:
We don't need the `deleteContent` parameter here

Reply:
Nice catch